packages/thirdweb/src/wallets/wallet-connect/types.ts

```diff
@@ -181,6 +181,8 @@ export type WCAutoConnectOptions = {
   savedConnectParams?: SavedConnectParams;
 
   chain?: Chain;
+
+  onDisplayUri?: (uri: string) => void;
```

Comment:
Does this need docs for what it is? (not a mobile dev, not sure if its obvious or not)

Reply:
I actually dont need this anymore, removing